docs/iris/example_code/Meteorology/COP_1d_plot.py

```diff
@@ -81,8 +81,7 @@ def main():
                 label='pre-industrial', lw=1.5)
 
     # Establish where r and t have the same data, i.e. the observations
-    common = np.where(a1b_mean.data == e1_mean.data)[0]
-    observed = a1b_mean[common]
+    observed = a1b_mean[:np.argmin(np.isclose(a1b_mean.data, e1_mean.data))]
```

Comment:
Hi @marqh sorry to interfere here.  (My comment below  has nothing to do with this code in particular.)

I agree that `np.isclose` might be what people want there, we rarely want `==` when dealing with float arrays.

But `np.where` with a variable named as `common` used as `index` is much clearer than `:np.argmin`.